.github/workflows/tests.yml

```diff
@@ -1,6 +1,6 @@
 name: Node CI
 
-on: [push]
+on: [push, pull_request]
```

Comment:
This will run when the PR is created and on every push to it after creation? Or only when created?

Reply:
from https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

> Runs your workflow when activity on a pull request in the workflow's repository occurs. For example, if no activity types are specified, the workflow runs when a pull request is opened or reopened **or when the head branch of the pull request is updated**.

Reply:
this will run a workflow action when a PR is created, irrespective of the author's membership.

Reply:
So, I pushed another commit to test if that would also create a new CI job, and it did. 
Looks like this will work just as desired.

Reply:
> or when the head branch of the pull request is updated.
Perfect